docs/introduction.rst

```diff
@@ -12,15 +12,27 @@ Dependencies
 Data levels
 ===========
 
-* R0
-* R1
-* DL1a
-* DL1b
-* DL2
-* DL3
++------+---------------------------------------------------------------------------+-------------+
+| Level| Description                                                               | File Format |
++======+===========================================================================+=============+
+| R0   | Uncalibrated RAW waveforms from the camera                                | ZFITS       |
++------+---------------------------------------------------------------------------+-------------+
+| R1   | Calibrated waveforms from the camera                                      |             |
++------+---------------------------------------------------------------------------+-------------+
+| DL1a | Integrated charge and peak position of the waveform                       | HDF5        |
++------+---------------------------------------------------------------------------+-------------+
+| DL1b | Image parameters (width, length, intensity, etc.)                         | HDF5        |
++------+---------------------------------------------------------------------------+-------------+
+| DL2  | Event parameters (energy, direction, time, etc.)                          | HDF5        |
++------+---------------------------------------------------------------------------+-------------+
+| DL3  | Lists of reconstructed events after event selection                       | FITS        |
+|      | with corresponding IRFs (PSF, EDISP, BKG, etc.)                           |             |
```

Comment:
Instead of BKG (which we are currently not producing as an IRF in the standard analysis), we should mention AEFF. 
The ordering may be (AEFF, EDISP, PSF, etc) to also indicate that the first 2 IRFs are always created, but others are optional. 
Not necessary to show this intention here, but would be good to avoid any misunderstanding general analyzers may have regarding the IRFs generated by lstchain.

Reply:
And maybe, we can also arrange the analysis step for the DL2 to DL3 stage to again, avoid any misunderstandings for general analyzers, to be in the correct order -
``lstchain.tools.lstchain_create_irf_files``
``lstchain.tools.lstchain_create_dl3_file``
``lstchain.tools.lstchain_create_dl3_index_files``

Reply:
Made the modifications.